server/src/db.ts

```diff
@@ -158,7 +163,7 @@ export async function addExternalIds(
 export async function updateLocation(
   location: ProviderLocation,
   data: any,
-  { mergeSubfields = true } = {}
+  { mergeSubfields = true, source = "" } = {}
```

Comment:
Seems useful to set a default here?

Reply:
Hmmmm, like what? This is the source from the loader (maybe I should call it `loaderSource`?), so I’m not sure how we’d pick any reasonable default, other than "null" or "?" or something.

Reply:
Maybe `"unknown"`? Just trying to avoid `source:` showing up somewhere accidentally.

Reply:
Done.